resources/styles/components/learning-guide.less

```diff
@@ -0,0 +1,239 @@
+@guide-width: 1270px;
+@guide-height: 390px;
+
+.learning-guide {
+  width: 100%;
+  background-repeat: repeat;
+  position: relative;
+
+  .x-axis-bg-repeat {
+    // global body padding-left/right prevents line from reaching ends at the moment...body classes with component name?
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 41px;
+    background: @tutor-guide-green;
+    z-index: -1;
+  }
+  .learning-guide-chart {
+    width: @guide-width;
+    height: @guide-height;
+    margin: 0 auto;
+
+    svg {
+      width: @guide-width;
+      height: @guide-height;
+
+      .main-title {
+        font-size: .13em;
+        font-family: Georgia;
```

Comment:
Can you move this font into `typography.less` and refer to it here with a mixin defined in `typography.less`?